scenes/cslib_string.txt

```diff
@@ -291,4 +291,39 @@
 	*set char_list_index char_list_len
 	*goto contains_loop
 *set cslib_ret false
-*return
\ No newline at end of file
+*return
+
+
+*comment CONSISTS_OF
+*comment ------------------------------
+*comment Checks if the given string consists
+*comment of any characters in a given list.
+*comment ------------------------------
+*comment 		params:
+*comment 			p_string(string): a string
+*comment 			p_char_list(string): a string of characters
+*comment 		returns:
+*comment 			(boolean): true if p_string consists only of chars in char_list, else false
+*label consists_of
+*params p_string p_char_list
+*set cslib_ret true
+*if (p_char_list = "")
+	*bug Character list can not be empty.
```

Comment:
Something like:

*bug cslib_string.consists_of: Character list must not be empty.

Since CS doesn't have a stack trace, we need to give the user as much help as possible when things go boom.